bitvision/engine/data_bus.py

```diff
@@ -5,8 +5,9 @@
 
```

Comment:
I'm not accepting any of the changes in this file. You accidentally deleted a lot of important code.

Reply:
We walked through that file together and made those changes on a video call. I deleted those bc you said to. I wouldn't have touched any of that otherwise.

Reply:
We didn't make these changes together. Why would we delete any of this code? I think you may not have been up-to-date with master when you made these changes, see the below screenshots:

<img width="530" alt="screen shot 2018-07-02 at 10 28 05 pm" src="https://user-images.githubusercontent.com/18684735/42195563-611fbd3a-7e47-11e8-953c-13b473ee7790.png">
<img width="1066" alt="screen shot 2018-07-02 at 10 28 22 pm" src="https://user-images.githubusercontent.com/18684735/42195564-612c0450-7e47-11e8-9053-f1df6a97d577.png">

Reply:
Fixed.